lib/matplotlib/tests/test_collections.py

```diff
@@ -624,3 +624,14 @@ def test_lslw_bcast():
     col.set_linestyles(['-', '-', '-'])
     assert_equal(col.get_linestyles(), [(None, None)] * 3)
     assert_equal(col.get_linewidths(), [1, 2, 3])
+
+
+@image_comparison(baseline_images=['scatter_post_alpha'],
```

Comment:
Is this a test that can just be a "check the colours haven't changed" test instead of an image test?